shopfloor_reception/services/reception.py

```diff
@@ -1235,6 +1243,11 @@ def _post_shopfloor_created_line(self, selected_line):
         )
 
     def _auto_post_line(self, selected_line):
+        # If user only processed 1/5 and is the only one working on the move,
+        # then selected_line is the only one related to this move.
+        # In such case, we must ensure there's another move line with the remaining
+        # quantity to do, so selected_line is extracted in a new move as expected.
+        selected_line._split_partial_quantity()
```

Comment:
```suggestion
        new_move_line = selected_line._split_partial_quantity()
```
We should call unmark_move_line_as_picked and set the location_dest_id back to stock.move's location_dest_id
if a split was happening 
```
            stock = self._actions_for("stock")
            stock.unmark_move_line_as_picked(new_move_line)
            new_line.location_dest_id = new_move_line.move_id.location_dest_id
```

Reply:
https://github.com/camptocamp/wms/pull/118

Reply:
Hello @mt-software-de , I updated the branch, can you please rebase? I'm checking this.
Thanks!

Reply:
This is correct but you also need to recompute the putaway.
It is easier if you always set product_uom_qty to 0, then there is no move line backorder (i.e. nothing to split).
I would do:
```
if product_uom_qty:
    selected_line.product_uom_qty = 0
```
and drop the `_split_partial_quantity`

Reply:
> Hello @mt-software-de , I updated the branch, can you please rebase? I'm checking this. Thanks!

rebase was done